source/common/stats/symbol_table_creator.cc

```diff
@@ -3,22 +3,15 @@
 namespace Envoy {
 namespace Stats {
 
-bool SymbolTableCreator::initialized_ = false;
-bool SymbolTableCreator::use_fake_symbol_tables_ = false;
-
 SymbolTablePtr SymbolTableCreator::initAndMakeSymbolTable(bool use_fake) {
-  ASSERT(!initialized_ || (use_fake_symbol_tables_ == use_fake));
-  use_fake_symbol_tables_ = use_fake;
+  if (use_fake) {
+    ENVOY_LOG_MISC(warn, "Fake symbol tables have been removed. Please remove references to "
+                         "--use-fake-symbol-table");
```

Comment:
Can we open a tracking issue to remove this CLI option? We can clean this up after we cut the release.